tests/src/test/scala/whisk/core/controller/test/ActivationsApiTests.scala

```diff
@@ -292,27 +306,56 @@ class ActivationsApiTests extends ControllerTestCommon with WhiskActivationsApi
         end = Instant.now)
     }.toList
     activations foreach { put(activationStore, _) }
-    waitOnView(activationStore, namespace.root, 2, WhiskActivation.collectionName)
+
+    val activationsInPackage = (1 to 2).map { i =>
+      WhiskActivation(
+        namespace,
+        EntityName(s"xyz"),
+        creds.subject,
+        ActivationId(),
+        start = Instant.now,
+        end = Instant.now,
+        annotations = Parameters("path", s"${namespace.asString}/pkg/xyz"))
+    }.toList
+    activationsInPackage foreach { put(activationStore, _) }
+
+    waitOnView(activationStore, namespace.root, 4, WhiskActivation.view)
 
     whisk.utils.retry {
       Get(s"$collectionPath?name=xyz") ~> Route.seal(routes(creds)) ~> check {
         status should be(OK)
         val response = responseAs[List[JsObject]]
-        activations.length should be(response.length)
-        activations forall { a =>
+        val allActivations = activations ++ activationsInPackage
+        allActivations.length should be(response.length)
+        allActivations forall { a =>
           response contains a.summaryAsJson
         } should be(true)
```

Comment:
A check like this will result in `false was not true` which usually doesn't help much.

Would

```scala
response should contain theSameElementsAs allActivations.map(_.summaryAsJson)
```

work? (I think that might apply to lots of tests here, so it'd be okay not to do this on this PR)

Reply:
yes should work - i made this change in view tests but going to defer cleaning up the rest of theses in this package as there are many of them and we should change all of them.